src/main/java/kazoo/model/Partitura.java

```diff
@@ -15,6 +15,7 @@
     private String numerador;
     private String denominador;
     private String nombre;
+    private Boolean esPublica = false;
```

Comment:
esto me lo crea asi? not bad, pensé que para hacer esto había que ponerlo en el constructor